.processes/release.md

```diff
@@ -128,78 +128,89 @@ particular branch to deploy on every change.
 
    ```
-   mkdir -p packages/ethereum/deployments/${ENV}/xdai
-   cp packages/ethereum/deployments/${OLD_ENV}/xdai/* packages/ethereum/deployments/${ENV}/xdai/
-   cp packages/ethereum/deployments/${OLD_ENV}/xdai/.chainId packages/ethereum/deployments/${ENV}/xdai/
-   rm packages/ethereum/deployments/${ENV}/xdai/HoprChannels.json
+   mkdir -p packages/ethereum/deployments/${RELEASE_NAME}/xdai
+   cp packages/ethereum/deployments/${ENVIRONMENT_NAME}/xdai/* packages/ethereum/deployments/${RELEASE_NAME}/xdai/
+   cp packages/ethereum/deployments/${ENVIRONMENT_NAME}/xdai/.chainId packages/ethereum/deployments/${RELEASE_NAME}/xdai/
+   rm packages/ethereum/deployments/${RELEASE_NAME}/xdai/HoprChannels.json
    ```
 
    NOTE: Don't include the deployment of HoprChannels, because this will be re-deployed anyway by the CD system.
-
    Changes should be committed locally.
 
-5. Delete all topology VM instances of ${OLD_OLD_RELEASE} and ${OLD_RELEASE} and all instances of ${OLD_OLD_RELEASE}, e.g. `prague-1.84`. Check `gcloud compute instance-groups managed list` for a list and delete the instance groups using
-   ```sh
-   gcloud compute instance-groups managed delete ${INSTANCE_GROUP_NAME} --region=$REGION
-   ```
-6. (on `release/${RELEASE_NAME}`) Now everything is ready and can be pushed to GitHub: `git push origin`. Wait until the deployment of the basic cluster has completed by the CD.
-7. Create a release tracking PR which can be used to follow CD builds. However, the PR should never be merged! As a reference take a look at https://github.com/hoprnet/hoprnet/pull/3048
-8. (on `release/${RELEASE_NAME}`) Start a topology cluster using the [script](./release.md#topology-deployment-script) mentioned at the end of this document.
-9. Create a release testnet page in the wiki at: https://www.notion.so/Testnets-e53255f7003f4c8eae2f1b6644a676e0
-   You may use previous testnet pages as templates. Ensure all started nodes are documented.
-10. Share the links to the release tracking issue, tracking PR and testnet wiki page in the `#release` Element channel.
+6. Delete the topology VM instances of the ${OLD_RELEASE_NAME}
+
+- Check `gcloud compute instance-groups managed list` for a list and delete the instance groups using
+
+```sh
+HOPRD_PERFORM_CLEANUP=true ./scripts/setup-gcloud-cluster.sh "${ENVIRONMENT_NAME}" "" "${OLD_RELEASE_NAME}-topology-1-91"
+```
+
+7. On the `release/${RELEASE_NAME}` branch, check that everything is ready and push it to GitHub by executing : `git push origin`. Wait until the [deployment of the cluster](https://github.com/hoprnet/hoprnet/actions/workflows/deploy.yaml) has finished successfully.
+8. Create a Pull Request for tracking the release changes against the `master` branch. Remark in the PR description that it should never be merged!. Also use the label `DO NOT MERGE` and `release`. As a reference take a look at https://github.com/hoprnet/hoprnet/pull/4311
```

Comment:
I would add, that a custom `release/<release_name>` tag might be created. This is then used to tag the merge-back PRs.